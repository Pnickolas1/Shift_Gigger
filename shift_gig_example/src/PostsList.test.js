import React from 'react';
import PostList from './components/Posts/PostList'
import renderer from 'react-test-renderer';

it('Jest - PostList snapshop renders correctly', () => {
  const tree = renderer
    .create(<PostList/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

