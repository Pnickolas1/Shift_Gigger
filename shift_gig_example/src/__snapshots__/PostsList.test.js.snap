// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest - PostList snapshop renders correctly 1`] = `<div />`;
